sources/vega/data/v1/data.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package data.v1;
```

Comment:
Could we make this package `vega.data.v1` to reflect what other packages are doing?

Reply:
Yes

Reply:
This has been chnaged here https://github.com/vegaprotocol/protos/pull/514/files#diff-e5814c05abaec52ff84d42ee4896683612257a404b721c65b14efe5febcb762e and everywhere needed